Runtime error with line #0

Not critical for sure but would be nice to have user friendly message
 
```
➜  test git:(master) ✗ expanderr ./test.go:#0
panic: runtime error: index out of range

goroutine 1 [running]:
main.parseQueryPos(0xc420056340, 0xc420098180, 0x7ffeefbff08a, 0xc, 0x0, 0x4, 0xc420098180, 0x0, 0x0, 0x1e)
	/Users/styx/Projects/go/src/github.com/stapelberg/expanderr/expanderr.go:75 +0x62b
main.logic(0x13f7920, 0xc42000c018, 0x14161e0, 0x7ffeefbff08a, 0xc, 0x0, 0x0)
	/Users/styx/Projects/go/src/github.com/stapelberg/expanderr/expanderr.go:415 +0x250
main.main()
	/Users/styx/Projects/go/src/github.com/stapelberg/expanderr/expanderr.go:683 +0xbd
```

--- a/pos.go
+++ b/pos.go
@@ -56,7 +56,7 @@ func parsePos(pos string) (filename string, startOffset, endOffset int, err erro
 		startOffset = parseOctothorpDecimal(offset[:comma])
 		endOffset = parseOctothorpDecimal(offset[comma+1:])
 	}
-	if startOffset < 0 || endOffset < 0 {
+	if startOffset < 1 || endOffset < startOffset {
 		err = fmt.Errorf("invalid offset %q in query position", offset)
 		return
 	}

